src/script/shortest-path.cc

```diff
@@ -19,8 +19,8 @@ void ShortestPath(const FstClass &ifst, MutableFstClass *ofst,
 }
 
 REGISTER_FST_OPERATION(ShortestPath, StdArc, ShortestPathArgs);
-//REGISTER_FST_OPERATION(ShortestPath, LogArc, ShortestPathArgs);
-//REGISTER_FST_OPERATION(ShortestPath, Log64Arc, ShortestPathArgs);
+REGISTER_FST_OPERATION(ShortestPath, LogArc, ShortestPathArgs);
+REGISTER_FST_OPERATION(ShortestPath, Log64Arc, ShortestPathArgs);
```

Comment:
LGTM, but I am wondering why did you have to comment these lines back in the next commit in this MR. Is this intentional?

The log weight naturally does not have the path property... Seems like MSVC is too eager to evaluate the condition in the static_assert again, like it did with the `enable_if`!

Reply:
Feel free to merge if that's intended. Are you able to merge in this repo?